hw/top_darjeeling/data/autogen/top_darjeeling.gen.hjson

```diff
@@ -9258,6 +9258,15 @@
           expose: "true"
           name_top: RvCoreIbexPipeLine
         }
+        {
+          name: TLRsvdInitiatorId
+          desc: TLUL user bit vector set on outgoing transfers
```

Comment:
I'm not sure I know what "bit vector set" means. Is this a bit vector that is set? Maybe worth rephrasing?

Reply:
I just rephrased that sentence.